### PR TITLE
Added field to db

### PR DESCRIPTION
**Pull request by Ixbitz**
Easy peasy lemon Squeezy

### CODE REVIEW

File: routes/api/bar.ts [hunk 1]
```diff
@@ -27,13 +27,15 @@ router.post('/', oauth.authenticate({scope:"bar"}), function(req, res) {
     let errorMessage: String = '';
     if (!req.body.name) errorMessage += 'Please fill in your bar name';
     if (!req.body.description) errorMessage ? errorMessage += ', bar description' : errorMessage = 'Please fill in your bar description';
-    if (!req.body.location) errorMessage ? errorMessage += ' and address' : errorMessage = 'Please fill in your bar address';
+    if (!req.body.city) errorMessage ? errorMessage += ' , city' : errorMessage = 'Please fill in your bar address';
```
Comment (C-Alexander) [inline routes/api/bar.ts]: Isn't this meant to be City?

**Review by C-Alexander — CHANGES_REQUESTED** [review]
Fix the text and for the rest it's fine.

**Comment by Ixbitz** [comment]
Fixed

**Review by C-Alexander — APPROVED** [review]
lkllk

**Pull request merged**